mmrotate/evaluation/functional/bbox_overlaps.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmcv.ops import box_iou_rotated
+from torch import Tensor
+
+from mmrotate.structures.bbox import RotatedBoxes
+
+
+def rbbox_overlaps(bboxes1: Tensor,
```

Comment:
the bbox_overlaps in mmrotate/evaluation/functional/bbox_overlaps.py should be numpy version, the tensor version should be put in mmrotate/structures/bbox/bbox_overlaps.py